googleapis_auth/CHANGELOG.md

```diff
@@ -8,6 +8,11 @@
 - `authenticatedClient` function: added optional `bool closeUnderlyingClient`
   parameter.
 
+#### `auth_io.dart`
```

Comment:
nit – put this *below* browser – so it's sorted!

Reply:
oh sorry. I changed